pyTigerGraph/gds/gsql/dataloaders/edge_loader.gsql

```diff
@@ -35,6 +35,9 @@ CREATE QUERY edge_loader_{QUERYSUFFIX}(
 
     # Shuffle vertex ID if needed
     start = {ANY};
+    IF filter_by IS NOT NULL THEN
```

Comment:
I thought this is removed? because there is filtering by edge on line 60

Reply:
Good catch - I only removed it in the edge neighbor loader